packages/flutter/lib/src/material/chip.dart

```diff
@@ -74,6 +78,18 @@ class Chip extends StatelessWidget {
   /// The delete button is included in the chip only if this callback is non-null.
   final VoidCallback onDeleted;
 
+  /// The style to be used for the label text.
+  final TextStyle labelStyle;
+
+  /// Message for the chip delete button's tooltip.
```

Comment:
these docs should reference onDeleted and say that it's ignored if that's null.
￼